.vscodeignore

```diff
@@ -15,6 +15,7 @@ gulp*
 node_modules/**
 out/**
 resources/readme/**
+resources/walkthroughs/*.gif

```

Comment:
Additional note, we're not including the GIFs in the bundle because they would dramatically increase the VSIX size (like, 2 MB -> 10 MB). Since VSIX size is inextricably tied to update velocity--if they activate the extension before download is complete, the update gets delayed to the next session--I want to ensure that the VSIX size stays as low as possible.

Reply:
@bwateratmsft so then what will we do instead?

Reply:
@ucheNkadiCode they get loaded directly from GitHub (i.e. from the web instead of local filesystem)